spec/support/seeds/seeds_e2e.rb

```diff
@@ -65,7 +65,7 @@
 
 seeder.create_if_doesnt_exist(User, "email", "trusted-user-1@forem.local") do
   user = User.create!(
-    name: "Trusted User 1",
+    name: "Trusted User 1 \\:/",
```

Comment:
These changes helped me surface failing tests that only Cypress caught.  Looking at this, I'm considering adding quotes around one of these users.